etna/lib/etna/spec/vcr.rb

```diff
@@ -0,0 +1,88 @@
+require 'webmock/rspec'
+require 'vcr'
+require 'openssl'
+require 'digest/sha2'
+require 'base64'
+
+def setup_base_vcr(spec_helper_dir)
+  VCR.configure do |c|
+    c.hook_into :webmock
+    c.cassette_serializers
+    c.cassette_library_dir = ::File.join(spec_helper_dir, 'fixtures', 'cassettes')
+    c.allow_http_connections_when_no_cassette = true
+
+    c.register_request_matcher :try_body do |request_1, request_2|
+      if request_1.headers['Content-Type'].first =~ /application\/json/
+        if request_2.headers['Content-Type'].first =~ /application\/json/
+          JSON.parse(request_1.body) == JSON.parse(request_2.body)
```

Comment:
Hm....so this doesn't cause an error in CI, but when I test locally, I'm getting the following exception:

```
Failures:

  1) Polyphemus::MetisFileEtl should process metis files, and support reset
     Failure/Error: metis_client = Etna::Clients::Metis.new(host: 'https://metis.development.local', token: ENV['TOKEN'] || 'test-token', persistent: false)
     
     JSON::ParserError:
       765: unexpected token at ''
     # /etna/lib/etna/spec/vcr.rb:19:in `block (2 levels) in setup_base_vcr'
```

This is caused when trying to match on the `options` request to set up the Metis client, because the body is an empty string.

```
[1] pry(main)> request_1
=> #<struct VCR::Request
 method=:options,
 uri="https://metis.development.local/",
 body="",
 headers=
  {"Content-Type"=>["application/json"],
   "Accept"=>["application/json, text/*"],
   "Authorization"=>["Etna test-token"],
   "Accept-Encoding"=>["gzip;q=1.0,deflate;q=0.6,identity;q=0.3"],
   "User-Agent"=>["Ruby"]}>
[3] pry(main)> request_2
=> #<struct VCR::Request
 method=:options,
 uri="https://metis.development.local/",
 body="",
 headers=
  {"Content-Type"=>["application/json"],
   "Accept"=>["application/json, text/*"],
   "Authorization"=>["<AUTHORIZATION>"],
   "Accept-Encoding"=>["gzip;q=1.0,deflate;q=0.6,identity;q=0.3"],
   "User-Agent"=>["Ruby"]}>
[4] pry(main)> JSON.parse(request_1.body)
JSON::ParserError: 765: unexpected token at ''
from /usr/local/lib/ruby/2.5.0/json/common.rb:156:in `parse'
```

Should this stick to [the previous way of checking the matching requests](https://github.com/mountetna/monoetna/blob/21a4c8e56c94bbc5a58e7ee96354fc77d9e90b48/polyphemus/spec/spec_helper.rb#L41-L46)?

Reply:
Yeah, I'll revert to that logic.